Tools/Scripts/libraries/webkitbugspy/webkitbugspy/mocks/bugzilla.py

```diff
@@ -166,6 +166,8 @@ def _issue(self, url, id, credentials=None, data=None):
                 issue['related']['regressed_by'] = data['regressed_by']
             if data.get('regressions'):
                 issue['related']['regressions'] = data['regressions']
+            if data.get('see_also'):
+                issue['references'] += data['see_also']['add']
```

Comment:
As above, I would've expected similar behaviour to `keywords` (now below).

Reply:
As above (:P), there's no `set` for `see_all`. Then, the logic is essentially the same since we don't need to condition for different API behaviour.